src/view/cards/wdr/wdr_content_reco_card.py

```diff
@@ -124,9 +124,18 @@ def draw(self, content_dto: WDRContentItemDto, nr, model, model_config, modal_fu
 
         child_objects = [
             teaserimage,
-            pn.pane.Markdown(f""" ### Score: {str(round(content_dto.dist, 2))}"""),
         ]
 
+        dto_score_attribute = None
+        if hasattr(content_dto, "dist"):
```

Comment:
fühlt sich etwas hacky an, können wir nicht einfach ein einheitliches Feld für den Score einführen?

Reply:
Das Problem ist hier, dass der score als score beim Request gegen den PA Service mitkommt und das content Item den aber gar nicht aufnehmen kann, da es bei der Suche nach Recommendations über den OSS gar kein Feld "dist" hat (initial). Das wird bei der Berechnung in das Content Item rein geschrieben und geht also an der Verarbeitung über den PA Service vorbei. Das Mapping setzt dafür auch zu spät an, das Problem ist da, das der score eben direkt durch die response kommt und dann auch direkt aufgenommen werden muss.

Reply:
So, hab die Stelle gefunden, jetzt ist es richtig.

Reply:
Sehr nice, is jetzt viel weniger Code und Logik. I like!